### ISSUE TITLE
Vomnibar is not shown completely when `browser.display.os-zoom-behavior` is set to 2 in Firefox

### ISSUE THREAD

**Issue by MahdiNazemi** [gdh1995/vimium-c#887]
**Describe the bug**

Firefox has an option to increase the font size on web pages without increasing the size of images, UI elements, etc. This can be achieved by setting `browser.display.os-zoom-behavior` to 2 as explained [here](https://www.reddit.com/r/firefox/comments/vnpip6/comment/ie9zzoe/?context=3).

Setting this options causes the Vomnibar to be shown in a scrollable area (see the right side of the screenshot):

<img width="2547" alt="Screenshot 2023-03-17 at 12 17 35 PM" src="https://user-images.githubusercontent.com/11276349/225999902-b6574af5-a6a2-40fc-ade8-8a9d0360e25a.png">

Can I customize the Vomnibar's CSS to resolve this issue?

**To Reproduce**

1. Navigate to about:config in Firefox.
2. Set `browser.display.os-zoom-behavior` to 2.
3. Open Vomnibar

**Browser, OS and Vimium C versions**

- Browser name: Firefox
- Browser version: 111.0
- Vimium C version: 1.99.97
- OS name and version: macOS Ventura (13.2.1)

**Comment by gdh1995** [comment]
Sorry I can not reproduce it on Firefox 110 + Win 11 + DPI=200%. The `os-zoom-behavior` seems to affect nothing on my computer.

**Comment by MahdiNazemi** [comment]
Is there any additional information I can provide to help?

Here are two other screenshots from the exact same location in my display, one with `browser.display.os-zoom-behavior` set to 1 and the other to 2.

`browser.display.os-zoom-behavior = 2`:
<img width="1292" alt="Screenshot 2023-03-19 at 10 24 48 AM" src="https://user-images.githubusercontent.com/11276349/226195137-7e1378d0-8384-4dc7-889f-3a979f38e306.png">

`browser.display.os-zoom-behavior = 1`:
<img width="1292" alt="Screenshot 2023-03-19 at 10 25 09 AM" src="https://user-images.githubusercontent.com/11276349/226195141-095c7d58-c496-4045-a709-0e0d4fbf79b3.png">

**Comment by MahdiNazemi** [comment]
I changed the first number to 87 and 107, but none of them fixed the issue.

**Comment by MahdiNazemi** [comment]
You are right. I may have forgotten to reload the page. I am very sorry about that.

While 87 and 97 have the same issue, setting the height to 107 resolves the issue.

<img width="1440" alt="Screenshot 2023-03-22 at 9 13 48 AM" src="https://user-images.githubusercontent.com/11276349/226968768-28d69d4d-d900-4a38-bf8b-d113f8eb7d21.png">

**Comment by MahdiNazemi** [comment]
> Have you added any custom CSS rules to Vomnibar?

No, I haven't.

> Could you move mouse pointer to the `<iframe>` line, so that the whole iframe area will be highlighted, and then make a screen snapshot again? It will help to find why `97` is not high enough.

Now, I am getting a different behavior. For the small Vomnibar screenshot I shared in my previous comment, 97 works fine too. But for a larger Vomnibar, neither 97 nor 107 work.

Here are a few screenshots with different settings.

**Height: 97; short Vomnibar**

<img width="1440" alt="Screenshot 2023-03-22 at 10 34 08 AM" src="https://user-images.githubusercontent.com/11276349/226990789-748a7ce2-1e6f-44f0-a0c0-d5cdccba68d2.png">

**Height: 97; tall Vomnibar (scrollbar is visible in the first screenshot)**

<img width="1440" alt="Screenshot 2023-03-22 at 10 34 42 AM" src="https://user-images.githubusercontent.com/11276349/226990985-20ee9f32-ea8a-475d-a05e-312b59b01ce3.png">

<img width="1440" alt="Screenshot 2023-03-22 at 10 34 54 AM" src="https://user-images.githubusercontent.com/11276349/226991029-87dc4896-ba17-4571-8157-b37b8ef4221c.png">

**Height: 107; short Vomnibar**

<img width="1440" alt="Screenshot 2023-03-22 at 10 36 58 AM" src="https://user-images.githubusercontent.com/11276349/226991253-2c1ba876-0a04-4617-a27f-b45c23563ec5.png">

**Height: 107; tall Vomnibar**

<img width="1440" alt="Screenshot 2023-03-22 at 10 37 17 AM" src="https://user-images.githubusercontent.com/11276349/226991289-e2e1d9a2-d32c-4380-a27d-abf07b0cce45.png">

**Comment by gdh1995** [comment]
Try this:
```
 "sizes": "88,3,50,0.8"
```

**Comment by MahdiNazemi** [comment]
> Try this:
> 
> ```
>  "sizes": "88,3,50,0.8"
> ```

Thank you. This works fine.

I will keep the issue open a bit more to see the behavior for different Vomnibar heights and close it if it works in all conditions.

**Comment by gdh1995** [comment]
The height number in `style` is about `sizes[0] + (N-1) * sizes[2]`, so appearantly a fixed `sizes[0]` is not enough. And, `50` will help a lot to auto fit different suggestion numbers.

While it's still strange that the default `44` gets not enough.

**Comment by gdh1995** [comment]
what's your screen's pixel resolution and font scale ratio?

**Comment by MahdiNazemi** [comment]
Screen Resolution: 1440x900 (Retina)

Firefox Settings:
Default Zoom: 120%
`browser.display.os-zoom-behavior = 2`
`ui.textScaleFactor = 115`
